File: CSharp/SpaceVIL/SpaceVIL/Items/WindowAnchor.cs
using System.Drawing;
using SpaceVIL.Core;

namespace SpaceVIL
{
    /// <summary>
    /// WindowAnchor is class representing the draggable window type of an item. 
    /// Supports all events except drag and drop despite that this class is draggable type.
    /// </summary>
    public class WindowAnchor : Prototype, IWindowAnchor
    {
        static int count = 0;
        /// <summary>
        /// Default WindowAnchor constructor.
        /// </summary>
        public WindowAnchor()
        {
            SetItemName("WindowAnchor_" + count);
            count++;
        }
    }
}

File: Java/SpaceVIL/spacevil/src/main/java/com/spvessel/spacevil/WindowAnchor.java
package com.spvessel.spacevil;

import com.spvessel.spacevil.Core.IWindowAnchor;

/**
 * WindowAnchor is class representing the draggable window type of an item.
 * Supports all events except drag and drop despite that this class is draggable
 * type.
 */
public class WindowAnchor extends Prototype implements IWindowAnchor {
    private static int count = 0;

    /**
     * Default WindowAnchor constructor.
     */
    public WindowAnchor() {
        setItemName("WindowAnchor_" + count);
        count++;
    }
}